src/components/HeroSection.vue

```diff
@@ -75,79 +71,69 @@ export default {
 <style lang="scss" scoped>
 @import '../../node_modules/foundation-sites/scss/foundation';
 
+h1 {
+  font-family: Roboto;
+  font-size: 2.375em;
+  color: #333333;
```

Comment:
should we use variables for colors?

Reply:
This isn't standard yet on CC Search. We'll migrate to using Vocabulary's variables